swap/peer.go

```diff
@@ -154,16 +169,18 @@ func (p *Peer) createCheque() (*Cheque, error) {
 // To be called with mutex already held
```

Comment:
Could you update the comment, mentioning the added functionality?

Reply:
done